doc/guides/onboarding-extras.md

```diff
@@ -24,7 +24,17 @@ request.
 * `tsc-agenda` - Open issues and pull requests with this label will be added to
   the Technical Steering Committee meeting agenda
 
---
+---
+
+* `author-ready` - A pull request is _author ready_ when:
+  * There is a CI run in progress or completed.
+  * There is at least one Collaborator approval.
+  * There are no outstanding review comments.
+
+Please always add the `author ready` label to the pull request in that case.
+Please always remove it again as soon as the conditions are not met anymore.
```

Comment:
I would clarify here that the label should be removed if the CI run fails

Reply:
makes sense! 👍 will add that clarification 😊